### PR TITLE
Add r-portfolioanalytics

### PR DESCRIPTION
**Pull request by janjagusch**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [ ] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [ ] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/r-portfolioanalytics/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    - r-zoo
+
+test:
+  commands:
+    - $R -e "library('PortfolioAnalytics')"           # [not win]
+    - "\"%R%\" -e \"library('PortfolioAnalytics')\""  # [win]
+
+about:
+  home: https://github.com/braverock/PortfolioAnalytics
+  license: GPL-3
```
Comment (janjagusch) [inline recipes/r-portfolioanalytics/meta.yaml:65]:
```suggestion
  license: GPL-3.0-or-later
```

File: recipes/r-portfolioanalytics/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+    - r-zoo
+
+test:
+  commands:
+    - $R -e "library('PortfolioAnalytics')"           # [not win]
+    - "\"%R%\" -e \"library('PortfolioAnalytics')\""  # [win]
+
+about:
+  home: https://github.com/braverock/PortfolioAnalytics
+  license: GPL-3.0-or-later
```
Comment (mfansler) [inline recipes/r-portfolioanalytics/meta.yaml:66]:
```suggestion
  license: GPL-3.0-only
```

File: recipes/r-pso/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    - r-base
+
+test:
+  commands:
+    - $R -e "library('pso')"           # [not win]
+    - "\"%R%\" -e \"library('pso')\""  # [win]
+
+about:
+  home: https://CRAN.R-project.org/package=pso
+  license: LGPL-3.0-or-later
```
Comment (mfansler) [inline recipes/r-pso/meta.yaml:43]:
```suggestion
  license: LGPL-3.0-only
```

File: recipes/r-roi.plugin.symphony/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+    - r-slam
+
+test:
+  commands:
+    - $R -e "library('ROI.plugin.symphony')"           # [not win]
+    - "\"%R%\" -e \"library('ROI.plugin.symphony')\""  # [win]
+
+about:
+  home: http://roi.r-forge.r-project.org/, https://r-forge.r-project.org/projects/roi/
+  license: GPL-3.0-or-later
```
Comment (mfansler) [inline recipes/r-roi.plugin.symphony/meta.yaml:49]:
```suggestion
  license: GPL-3.0-only
```

File: recipes/r-rsymphony/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+
+test:
+  commands:
+    - $R -e "library('Rsymphony')"           # [not win]
+    - "\"%R%\" -e \"library('Rsymphony')\""  # [win]
+
+about:
+  home: https://R-Forge.R-project.org/projects/rsymphony, https://projects.coin-or.org/SYMPHONY,
+    https://www.coin-or.org/download/source/SYMPHONY/
+  license: EPL-1.0
```
Comment (mfansler) [inline recipes/r-rsymphony/meta.yaml:58]: A copy of this license needs to be included. https://spdx.org/licenses/EPL-1.0.html

**Review by mfansler — CHANGES_REQUESTED** [review]
some license touch ups

**Comment by janjagusch** [comment]
> some license touch ups

Thanks for the review, @mfansler. The whole stack depends on https://github.com/coin-or/SYMPHONY, which is not yet on CF. I will close this PR until we have `symphony` on CF.

**Pull request closed without merging**